packages/gasket-plugin-nextjs/lib/index.js

```diff
@@ -137,7 +137,15 @@ module.exports = {
         // interact with the express router we want to add a last, catch all
         // route that will activate the `next`.
         //
-        expressApp.all('*', app.getRequestHandler());
+        const nextHandler = app.getRequestHandler();
+        expressApp.all('*', async (req, res, next) => {
```

Comment:
This will need to work for Fastify apps as well

Reply:
Just updated the fastify hook to use the nextPreHandling lifecycle as well. 👍🏻